tests/scripts/task_kg_test.sh

```diff
@@ -34,10 +34,10 @@ conda activate ${DGLBACKEND}-ci
 # test
 if [ "$2" == "cpu" ]; then
     pip3 uninstall dgl
-    pip3 install dgl==0.4.3post2
+    pip3 install dgl
 else
     pip3 uninstall dgl
-    pip3 install dgl-cu102==0.4.3post2
```

Comment:
Please add a comment about what is the latest dgl version that works with this patch. As we are using dgl._deprecate.graph.DGLGraph, this interface may be deleted in some date.

Reply:
Thank you @classicsong , I have added a minimum and maximum dependency for the tests

Reply:
This breaks the `partition` method because `dgl` switched the attribute name from `dgl.transform` to `dgl.transforms` in version `0.8`.

Reply:
https://github.com/dmlc/dgl/releases/tag/0.8.0 under `Graph Dataset and Transforms` section

Reply:
I'm using version 0.9.1 of dgl and got this error. A reasonable work around for this would be to switch using the Version class in `packaging.version.Version` on version `0.8`. I'll open a CR for this, (unless someone from this project indicates a preferred alternate way or says this shouldn't be done).